jdl/validators/jdl-with-application-validator.js

```diff
@@ -52,6 +52,11 @@ function createValidator(jdlObject, logger = console) {
         checkForErrors: () => {
             checkForApplicationErrors();
             jdlObject.forEachApplication(jdlApplication => {
+                const blueprints = jdlApplication.getConfigurationOptionValue('blueprints');
+                if (blueprints && blueprints.length > 0) {
+                    logger.warn('Generating application with blueprint, skipping jdl validation');
```

Comment:
This message implies the validator knows about the context where it's being used, meaning the validator knows we're using it to generate applications. It shouldn't.
What about something like "Blueprints are being used, the JDL validation phase is skipped."?

Reply:
ok with this